multik-api/src/test/kotlin/org/jetbrains/kotlinx/multik/ndarray/data/SliceTest.kt

```diff
@@ -53,4 +54,35 @@ class SliceTest {
         assertEquals(mk.ndarray(mk[mk[7]]), a[0..1, 1, 1, 1..2])
         assertEquals(mk.ndarrayOf(6), a[0..1, 1, 1, 0])
     }
+
+    @Test
+    fun testBase() {
+        val a = mk.ndarrayOf(0, 1, 2, 3, 4, 5)
+        val b = a[1..5] as NDArray
+        val c = a[1..3] as NDArray
```

Comment:
Why is `as` here?

Reply:
You are right, this cast in redundant.
Removed it in a0e6eb3